css/properties/aspect-ratio.json

```diff
@@ -69,7 +69,7 @@
               "version_added": false
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "14.5"
```

Comment:
The aspect ratio experimental feature is disabled for me on both macOS and iOS. Is this not the case for others?

Reply:
https://mdn-bcd-collector.appspot.com/tests/css/properties/aspect-ratio shows support for me on iOS 14.5, but feature detection can be broken.

Some of the basic test cases in https://wpt.live/css/css-sizing/aspect-ratio/ aren't passing, so I think most likely the property is recognized but the actual behavior is gated by a flag.

Reply:
I verified that this is just wrong.

Reply:
Addressed in 90183cb.

Reply:
And I've updated mdn-bcd-collector in a way to avoid the WebKit bug leading to this false positive in https://github.com/foolip/mdn-bcd-collector/pull/1132.